Description: Remove references to external images from docs
 Fetching resources from remote websites when viewing local documentation could
 breach privacy.
 It's fairly unlikely that anyone viewing this documentation cares about the
 current CI status.
Author: Stefano Rivera <stefanor@debian.org>
Forwarded: not-needed

--- a/README.rst
+++ b/README.rst
@@ -1,13 +1,6 @@
 Python Object Graphs
 ====================
 
-.. image:: https://travis-ci.org/mgedmin/objgraph.png?branch=master
-   :target: https://travis-ci.org/mgedmin/objgraph
-
-.. image:: https://coveralls.io/repos/mgedmin/objgraph/badge.png?branch=master
-   :target: https://coveralls.io/r/mgedmin/objgraph?branch=master
-
-
 ``objgraph`` is a module that lets you visually explore Python object graphs.
 
 You'll need `graphviz <http://www.graphviz.org/>`_ if you want to draw
